core/include/usServiceProperties.h

```diff
@@ -43,97 +41,6 @@ namespace us {
  */
 typedef std::unordered_map<std::string, Any> ServiceProperties;
```

Comment:
Is there a reason to keep this "usServiceProperties" class? Unless there is a reason to keep this typedef, I think it makes sense to remove this class and just use std::unordered_map. I can do this when I work on issue #85 (consolidating data types for bundle/service properties to use std::unordered_map).